tests/display/test_inputs.py

```diff
@@ -141,6 +164,7 @@ class TestScientificInput:
         [0, 0.01, 0.002]  # default #233: default <0.01 changes to 0
     ])
     def test_init_from_param(self, qtbot, min_, max_, default_value):
+        # First test is with value = default value
```

Comment:
as above, try adding a parametrize like @pytest.mark.parametrize("reference", ['min_', 'default']) and using `locals()[reference]` to access the appropriate value should work as expected.